### PR TITLE
Us119a coventry fake emd ref

### PR DESCRIPTION
**Pull request by MandyNimmo**
acceptance tests for us119a including new md ref in csv file

### CODE REVIEW

**Comment by flynnbops** [comment]
:+1:  LGTM
Will just have to make sure we keep track of any new data to be added to the CSV

File: features/step_definitions/borrower_views_deed.rb [hunk 1]
```diff
@@ -107,3 +107,57 @@
   page.should have_content("[Awaiting confirmation from #{f_name}"\
                              "#{m_name} #{s_name}]")
 end
+
+Given(/^I create a deed where the Lender is Coventry Building Society$/) do
+  step %(I have valid deed data with <#{borrower}> borrowers)
+  step %(I create the deed via the Deed API)
+  step %(the deed id is returned by the Deed API)
+end
```
Comment (nigelpain) [inline features/step_definitions/borrower_views_deed.rb]: This step should probably be removed as it is no longer being called from the scenario.

**Comment by Phemus** [comment]
:+1:

**Comment by nigelpain** [comment]
Just the one comment about the unused step, but once that's removed happy to :+1: this.

File: features/step_definitions/borrower_views_deed.rb (hunk 1 above)
Comment (MandyNimmo) [inline features/step_definitions/borrower_views_deed.rb]: Removed

**Pull request merged**